### PR TITLE
Review controller compaction

### PR DESCRIPTION
**Pull request by klanmiko**
Reduce the amount of json crud going to the frontend, DRY logic in skip and review route

### CODE REVIEW

File: src/controller/review.py [hunk 1]
```diff
@@ -34,13 +33,7 @@ def skip_application():
...
         abort(400, 'User is not currently assigned an application')
 
-    application = Application.get_application(current_user.get_id())
-    if application is None:
-        return Response(status=204)
-
-    answers = Answer.get_answers(application.id)
-    response = {'application': application, 'answers': answers}
-    return jsonify(Serializer.serialize_value(response))
+    return get_application()
```
Comment (thomasmunduchira) [inline src/controller/review.py]: 👍

File: src/controller/admin.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 login_manager = Shared.login_manager
 
 
-@admin.route('/api/admin/reload', methods=['POST'])
+@admin.route('/api/admin/reload', methods=['POST', 'GET'])
```
Comment (thomasmunduchira) [inline src/controller/admin.py]: This makes it easier for testing, but we should remove it when we get a working file upload solution.

File: src/models/Answer.py [hunk 3]
```diff
@@ -44,4 +46,11 @@ def get_answers_from_application(application):
     @staticmethod
     def get_answers(application_id):
         """Returns all answers associated with the application ID"""
-        return db.session.query(Answer).filter(Answer.application_id == application_id).all()
+        m_list = db.session.query(Answer) \
```
Comment (thomasmunduchira) [inline src/models/Answer.py]: Can we rename `m_list` to `answers`?

File: applications-frontend/src/components/Applicant.vue [hunk 4]
```diff
@@ -145,40 +145,30 @@ export default {
       this.answers = data.answers;
       this.application = data.application;
     },
+    handleResponseSuccess(response) {
```
Comment (thomasmunduchira) [inline applications-frontend/src/components/Applicant.vue]: 👍

File: src/controller/admin.py (hunk 2 above)
Comment (klanmiko) [reply src/controller/admin.py]: I think I committed this file by accident

File: src/models/Answer.py (hunk 3 above)
Comment (klanmiko) [reply src/models/Answer.py]: yea

File: src/controller/admin.py [hunk 5]
```diff
@@ -23,6 +23,8 @@ def reload():
         Answer.drop_rows()
         Application.drop_rows()
         Question.drop_rows()
+        #Question.question_enum.drop()
```
[the remark this answers is not in the document]
Comment (klanmiko) [reply src/controller/admin.py]: so what happened was the questiontype enum in the postgres database was out of sync with the sqlalchemy model

File: src/models/Answer.py [hunk 6]
```diff
@@ -44,4 +46,11 @@ def get_answers_from_application(application):
...
+        .filter(Answer.application_id == application_id) \
+        .join(Question) \
+        .filter(Question.question_type != QuestionType.ignore) \
```
[the remark this answers is not in the document]
Comment (klanmiko) [reply src/models/Answer.py]: ok

[the remark this answers is not in the document]
Comment (klanmiko) [reply src/models/Answer.py]: formatter didn't change anything

**Pull request merged**